### PR TITLE
fixed the types in the benchmarks

### PR DESCRIPTION
**Pull request by konnov**
This PR makes our performance benchmarks compatible with the new version of Apalache, which uses Snowcat

### CODE REVIEW

File: performance/Bakery-Boulangerie/APABakery.tla [hunk 1]
```diff
@@ -270,6 +281,73 @@ Inv == /\ TypeOK
                    \/ pc[nxt[i]] = "e3"
                 => max[nxt[i]] >= num[i]
              /\ (pc[i] = "cs") => \A j \in Procs \ {i} : Before(i, j)
+             
+FIndInv ==
```
Comment (shonfeder) [inline performance/Bakery-Boulangerie/APABakery.tla:285]: This doesn't look like just fixing the types. Is this an unrelated change?

File: performance/Bakery-Boulangerie/APABakery.tla [hunk 2]
```diff
@@ -270,6 +281,73 @@ Inv == /\ TypeOK
...
+w1(self) == /\ pc[self] = "w1"
+            /\ IF unchecked[self] # {}
+                  THEN /\ \E i \in unchecked[self]:
+                            nxt' = [nxt EXCEPT ![self] = i]
+                       /\ ~ flag[nxt'[self]]
+                       /\ pc' = [pc EXCEPT ![self] = "w2"]
+                  ELSE /\ pc' = [pc EXCEPT ![self] = "cs"]
+                       /\ nxt' = nxt
+            /\ UNCHANGED << num, flag, unchecked, max >>            
+*)
```
Comment (shonfeder) [inline performance/Bakery-Boulangerie/APABakery.tla:322]: Why the newly added commented out code?

File: performance/Bakery-Boulangerie/APABakery.tla [hunk 3]
```diff
@@ -470,6 +548,7 @@ ISpec == IInit /\ [][Next]_vars
              
 =============================================================================
 \* Modification History
+\* Last modified Wed Aug 14 13:37:20 CEST 2019 by tthai
```
Comment (shonfeder) [inline performance/Bakery-Boulangerie/APABakery.tla:551]: This line doesn't seem right

**Review by shonfeder — APPROVED** [review]
A couple questions, but otherwise lgtm.

File: performance/Bakery-Boulangerie/APABakery.tla (hunk 3 above)
Comment (konnov) [reply performance/Bakery-Boulangerie/APABakery.tla:556]: I think I copied it over from the OOPSLA artifact

File: performance/Bakery-Boulangerie/APABakery.tla (hunk 1 above)
Comment (konnov) [reply performance/Bakery-Boulangerie/APABakery.tla:290]: I guess, I will remove it. It comes from another artifact
